app/src/main/java/com/edulectronics/tinycircuit/Views/ExcerciseMenuActivity.java

```diff
@@ -0,0 +1,38 @@
+package com.edulectronics.tinycircuit.Views;
+
+import android.support.v7.app.AppCompatActivity;
+import android.os.Bundle;
+import android.view.View;
+import android.widget.LinearLayout;
+import android.widget.TextView;
+
+import com.edulectronics.tinycircuit.R;
+
+public class excerciseMenuActivity extends AppCompatActivity {
+    int n, excercise_amount = 10;
```

Comment:
Graag duidelijke variabele namen, of lokaal declareren als ze niet globaal nodig zijn. Is 'n' een counter voor een loop? Dan gewoon in de methode zelf declareren. Is 'n' een belangrijke teller voor iets die globaal bijgehouden moet worden? Dan zo noemen dat duidelijk is wat er geteld wordt.